setup.py

```diff
@@ -86,9 +86,9 @@
     install_requires=[
         'beautifulsoup4>=4.8.1,<4.13',
         'collatex>=2.2,<2.3',
-        'hdbscan>=0.8.20,<0.8.30',
+        'hdbscan>=0.8.20',
```

Comment:
Once `hdbscan` is working we will want to put an upper limit back on so dependabot can catch package updates and test them in the future.